kola/tests/misc/files.go

```diff
@@ -94,9 +94,19 @@ func sugidFiles(m platform.Machine, validfiles []string, mode string) error {
 func DeadLinks(c cluster.TestCluster) error {
 	m := c.Machines()[0]
 
-	output, err := m.SSH(`sudo find / /usr -ignore_readdir_race -xdev -xtype l`)
+	ignore := []string{
+		"/dev",
+		"/proc",
+		"/run/udev/watch",
+		"/sys",
+		"/var/lib/rkt",
```

Comment:
I presume /var/lib/docker should be excluded as well for the same reason as rkt. And why do /dev /proc /sys and /run need to be there if you are using -xdev?